src/EventuallyQueue.js

```diff
@@ -308,12 +308,14 @@ const EventuallyQueue = {
     }
     polling = setInterval(() => {
       const RESTController = CoreManager.getRESTController();
-      RESTController.ajax('GET', CoreManager.get('SERVER_URL')).catch(error => {
-        if (error !== 'Unable to connect to the Parse API') {
+      RESTController.request('GET', 'health/')
+        .then(() => {
           this.stopPoll();
           return this.sendQueue();
-        }
-      });
+        })
+        .catch(() => {
+          // 'Unable to connect to the Parse API' keep polling
```

Comment:
I don't mean to be pedantic, but would you mind catching the error and returning it `.catch(e => e);`, then checking for the status code to be 200? Empty `catch` blocks are just so hard to look at 😅

Reply:
Done! I like that `e => e` trick